### PR TITLE
Change api

### PR DESCRIPTION
**Pull request by banyan**
Add description for  and simplified api for consistency to other apis.

### CODE REVIEW

File: spec/sbpayment/configuration_spec.rb [hunk 1]
```diff
@@ -6,55 +6,27 @@
...
-        Sbpayment.config.disable_multiple_service_id
+  describe '#default_service_id' do
+    context 'when allow_multiple_service_id is true' do
+      before do
+        Sbpayment.config.allow_multiple_service_id = true
       end
 
-      it 'disables multiple_service_id mode' do
-        expect(Sbpayment.config.multiple_service_id?).to be(false)
+      it 'raises an error' do
```
Comment (kachick) [inline spec/sbpayment/configuration_spec.rb]: When described `'raises an error'` we do not expect specific error. That means matcher should be `raise_error(Exception)`.

File: lib/sbpayment/configuration.rb [hunk 2]
```diff
@@ -23,25 +23,10 @@ class Config
...
-    end
-
-    def multiple_service_id?
-      @multiple_service_id
-    end
+    attr_accessor :allow_multiple_service_id
 
     def default_service_id
-      if multiple_service_id?
+      if !!allow_multiple_service_id
```
Comment (kachick) [inline lib/sbpayment/configuration.rb]: `!!` is a hack to adjust method API. In this context, the idiom does't improve readability.

File: spec/sbpayment/configuration_spec.rb (hunk 1 above)
Comment (banyan) [inline spec/sbpayment/configuration_spec.rb]: do you mean `it 'raises Sbpayment::ConfigrationError' do` is also okay?

File: lib/sbpayment/configuration.rb [hunk 3]
```diff
@@ -23,25 +23,10 @@ class Config
     attr_accessor :cipher_code, :cipher_iv
     attr_accessor :hashkey
     attr_accessor :proxy_uri, :proxy_user, :proxy_password
-
-    def initialize
```
Comment (kachick) [inline lib/sbpayment/configuration.rb]: CRuby interpreter warns the access to undefined instance variables. I suggest to explicitly set instance variables in `#initialize`.

File: spec/sbpayment/configuration_spec.rb (hunk 1 above)
Comment (kachick) [inline spec/sbpayment/configuration_spec.rb]: Yes, I suggest `it 'raises a Sbpayment::ConfigrationError'` or `it 'raises a ConfigrationError'`.

File: lib/sbpayment/configuration.rb (hunk 3 above)
Comment (banyan) [inline lib/sbpayment/configuration.rb]: Curious! how can I see the warns?

```rb
class A
  def initialize
  end

  def foo
    @foo
  end
end

a = A.new
a.foo
```

File: spec/sbpayment/configuration_spec.rb (hunk 1 above)
Comment (banyan) [inline spec/sbpayment/configuration_spec.rb]: Fixed! a311de3

File: lib/sbpayment/configuration.rb (hunk 3 above)
Comment (kachick) [inline lib/sbpayment/configuration.rb]: When enabled `$VERBOSE = true`, It helps to avoid dangerous hike in Ruby.

Comment (banyan) [inline lib/sbpayment/configuration.rb]: Thanks! I haven't used it! FIxed! https://github.com/quipper/sbpayment.rb/commit/1a10ee62a11d18aecc1a17b7cc4110f976743bc5

File: README.md [hunk 4]
```diff
@@ -10,6 +10,34 @@ gem 'sbpayment'
 
 ## Usage
 
+### Example
+
+See [example](example).
+
+### Configure Options
```
Comment (kachick) [inline README.md]: Looks great to assist for beginners of this library! :tada:

**Comment by banyan** [comment]
Thanks! :relaxed:  @kachick

**Pull request merged**